packages/eslint-plugin/src/rules/unit-naming-rule.test.ts

```diff
@@ -8,61 +8,80 @@ const tester = new RuleTester({
   },
 })
 
-const ImportAtom = 'import {atom} from "@reatom/framework"'
-
 tester.run('unit-naming-rule', unitNamingRule, {
   valid: [
-    `${ImportAtom}; const some = atom(0, 'some')`,
+    `const some = atom(0, '_some')`,
+    `const some = action(0, 'some')`,
     {
-      code: `${ImportAtom}; const $some = atom(0, '$some')`,
+      code: `const $some = atom(0, '$some')`,
       options: [{ atomPrefix: '$' }],
     },
     {
-      code: `${ImportAtom}; const someAtom = atom(0, 'someAtom')`,
-      options: [{ atomPostfix: 'Atom' }],
+      code: `const someAtom = atom(0, '_someAtom')`,
+      options: [{ atomSuffix: 'Atom' }],
+    },
+    {
+      code: `function reatomSome() { const someAtom = atom(0, 'reatomSome.someAtom') }`,
+    },
+    {
+      code: `const Atoms = { someAtom: atom(0, 'Atoms.someAtom') }`,
+    },
+    {
+      code: `function reatomSome() { const Atoms = { someAtom: atom(0, 'reatomSome.Atoms.someAtom') } }`,
```

Comment:
Same.
```suggestion
      code: `function reatomSome() { const Atoms = { someAtom: atom(0, 'Some.Atoms.someAtom') } }`,
```

Btw, I really enjoy this feature, it will very helpfull in a complex busines code.